main/src/services/MonthsService.java

```diff
@@ -0,0 +1,54 @@
+package services;
+
+import models.Months;
+
+import java.util.Arrays;
+
+public class MonthsService {
+
+    public static void printHolidaysOfMonth(Months month) {
+        if (month == null) {
```

Comment:
Consider throwing IllegalArgumentException instead of returning
https://stackoverflow.com/questions/3881/illegalargumentexception-or-nullpointerexception-for-a-null-parameter/47710#47710